crates/stages/benches/criterion.rs

```diff
@@ -95,44 +96,58 @@ fn merkle(c: &mut Criterion) {
     // don't need to run each stage for that many times
     group.sample_size(10);
 
+    let db = setup::txs_testdata(DEFAULT_NUM_BLOCKS);
+
     let stage = MerkleStage::Both { clean_threshold: u64::MAX };
     measure_stage(
         &mut group,
+        &db,
         setup::unwind_hashes,
         stage,
-        1..DEFAULT_NUM_BLOCKS,
+        1..=DEFAULT_NUM_BLOCKS,
```

Comment:
was this a bug before?

Reply:
or well "bug", i guess this is in a benchmark, so doesn't rly matter

Reply:
the behavior didn't change: even though we were passing an exclusive range, we constructed `StageRange` struct with `range.start` and `range.end`